functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/customAssertions/ProjectReportAssert.java

```diff
@@ -0,0 +1,30 @@
+package org.arquillian.smart.testing.ftest.customAssertions;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.arquillian.smart.testing.ftest.testbed.project.Project;
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.Assertions;
+
+public class ProjectReportAssert extends AbstractAssert<ProjectReportAssert, Project> {
```

Comment:
I would welcome one assertion class `SmartTestingAssertions` that would delegate (base on the type) the other ones...

Reply:
BTW. You named the class `ProjectReportAssert`, but there is no assertion related to reports

Reply:
> I would welcome one assertion class SmartTestingAssertions that would delegate (base on the type) the other ones...

@MatousJobanek, Bartosz proposed against it, it's not really a value add for us, 
> It is a good idea, but not only for this reason - see here http://joel-costigliola.github.io/assertj/assertj-core-custom-assertions.html#single-assertion-entry-point.

Reply:
This is something different than I'm proposing - exactly as they write there:

> if you only have one entry point class for your custom assertions classes !

Let's do it as a pair programming task

Reply:
Ok, done using `SmartTestingSoftAssertions` as this class can be used as the main entry point.